### PR TITLE
Fix for Builder Id users Scan Popup.

### PR DESCRIPTION
**Pull request by ashishrp-aws**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

Fix for Builder Id users Scan Popup.

## Checklist
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt [hunk 1]
```diff
@@ -45,12 +45,10 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
         val actionManager = CodeWhispererExplorerActionManager.getInstance()
         val scanManager = CodeWhispererCodeScanManager.getInstance(project)
         actionManager.setMonthlyQuotaForCodeScansExceeded(false)
+        // Setting up listeners for Auto File Code Scan triggers.
```
Comment (ctlai95) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt:48]: nit: Does this comment need to be updated? It's not only for auto scan triggers

**Review by ctlai95 — APPROVED** [review]

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt (hunk 1 above)
Comment (ashishrp-aws) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt:48]: ah yes...will update the comment.

**Review by andrewyuq — APPROVED** [review]

**Review by rli — APPROVED** [review]
new changes are q-only and incorrectly refer to toolkit code, however we have not completed the separation yet so fine for now

**Comment by rli** [comment]
can we have a better description of what we are fixing and an appropriate changelog entry?

**Pull request merged**